SQRLPlatformAwareInstaller/Platform/Linux/Installer.cs

```diff
@@ -107,5 +114,40 @@ public DownloadInfo GetDownloadInfoForAsset(GithubRelease selectedRelease)
                 DownloadUrl = selectedRelease.assets.Where(x => x.name.Contains("linux-x64.zip")).First().browser_download_url
             };
         }
+
+
+        /// <summary>
+        /// Moves the Db from the current location to the new user space location
+        /// </summary>
+        /// <param name="currentPath"></param>
+        /// <returns></returns>
+        public bool MoveDb(string currentPath)
+        {
+            bool success = false;
+            if (!Directory.Exists(PathConf.ClientDBPath))
+            {
+                Directory.CreateDirectory(PathConf.ClientDBPath);
+                Inventory.Instance.AddDirectory(PathConf.ClientDBPath);
```

Comment:
Adding this call would be fatal!

This adds the whole DB directory to the install inventory, which would mean that the whole folder would be deleted upon uninstall. Keep in mind that the default folder in userspace is `My Documents\SQRL`, where for example all the identities of the GRC client are stored as well!

Also, in the uninstaller message, we specifically state that we don't touch the user data (like database etc.) upon unistallation!

So unless I'm missing something here, this call will get us into REAL troubles and definitely has to be removed!

Reply:
Nope you are right, FIxed